cmd/ocm-support/registryCredentials/cmd.go

```diff
@@ -0,0 +1,21 @@
+package registrycredentials
+
+import (
+	"ocm-support-cli/cmd/ocm-support/registryCredentials/delete"
+	"ocm-support-cli/cmd/ocm-support/registryCredentials/show"
+
+	"github.com/spf13/cobra"
+)
+
+// Cmd ...
+var Cmd = &cobra.Command{
+	Use:   "registryCredentials COMMAND",
+	Short: "Gets information about or execute actions on registry credentials.",
+	Long:  "Gets information about or execute actions on registry credentials.",
+	Args:  cobra.MinimumNArgs(1),
+}
```

Comment:
the `registryCredentials` command is quite long. We can make it easier to type by adding an alias. 

`Aliases: []string{"rcs"},`